lib/administrate/field/time.rb

```diff
@@ -0,0 +1,11 @@
+require_relative "base"
+
+module Administrate
+  module Field
+    class Time < Base
+      def default
+        data && data.strftime("%H:%M:%S")
```

Comment:
Ah, right. This is because Rails doesn't represent time without a date (this makes sense if you think about the concept of a specific time). So yes, we'd need a custom format. I'd suggest `time_without_date` or something.

@jcmorrow: "military time" is an amusing Americanism to me. 😛